src/runtime/fsys.r

```diff
@@ -2054,6 +2054,63 @@ function{0,1} seek(f,o)
 end
 
 #ifdef PosixFns
+#ifdef MSWindows
+int mswinsystem(char *s)
```

Comment:
Should probably move this to `rsys.r`. In general the `f` version of the file for the language visible functions, and the `r` file is for runtime support of those functions. eg. `fstruct.r -> `rstruct.r`,  `fsys.r` -> `rsys.r`.

Reply:
OK.